client/my-sites/themes/theme-showcase.jsx

```diff
@@ -138,6 +128,49 @@ class ThemeShowcase extends Component {
 		}
 	}
 
+	getTabTiers = () => {
+		const { translate } = this.props;
```

Comment:
Do you mean pass `props` as function argument, and we do:
```jsx
getTabTiers = ( { translate } ) => { ... }
```

Reply:
That's one way to do it. But I just mean that we either get from `this.props` or pass the `props` from the constructor consistently. Because currently we do:

```
this.tabTiers = this.getTabTiers( props );
this.tabFilters = this.getTabFilters( props );
```

Reply:
🤦🏻‍♂️ you're right, I didn't notice that I was already passing props.